### DISCUSSION TITLE
Tutorial sweep angle input

### DISCUSSION THREAD

**Discussion by askerp** [ImperialCollegeLondon/sharpy#163] [Q&A]
Hello,
While working with the 'Flutter Analysis of a Goland Wing using the SHARPy Linear Solver' tutorial, I noticed that the parameter 'sweep angle' has input in degrees and not in radians (which is mentioned in the documentation). 
Just wanted to bring it to the notice of the developers.

Thank you

**Comment by ACea15** [comment]
Hi,

That's not a problem, the model is built using the template ` flying_wings.py`, which internally converts the angles in degrees to radians.
